validation/views.py

```diff
@@ -380,6 +382,23 @@ def register(request):
 # TODO: Speaker bio page like https://ojibwe.lib.umn.edu/about/voices
 
 
+@require_http_methods(["POST"])
+def record_translation_judgement(request, phrase_id):
+    # TODO: check that user is logged in
+    phrase = get_object_or_404(Phrase, id=phrase_id)
+    print(phrase)
+    judgement = json.loads(request.body)
+
+    if judgement["judgement"] == "yes":
+        phrase.validated = True
+    elif judgement["judgement"] == "no":
+        phrase.validated = False
```

Comment:
```suggestion
    if judgement["judgement"] == "yes":
        phrase.validated = True
    elif judgement["judgement"] == "no":
        phrase.validated = False
    else:
        return HttpResponseBadRequest()
```

Defensive programming!